src/i18n/strings/en_EN.json

```diff
@@ -970,6 +971,7 @@
         "event_shield_reason_unverified_identity": "Encrypted by an unverified user.",
         "export_unsupported": "Your browser does not support the required cryptography extensions",
         "forgot_recovery_key": "Forgot recovery key?",
+        "identity_needs_reset_description": "You have to reset your cryptographic identity in order to ensure access to your message history",
```

Comment:
Perhaps?
```suggestion
        "identity_needs_reset_description": "You must reset your cryptographic identity in order to ensure you can access your message history",
```

Reply:
This is the wording that was provided in the Figma design